### PR TITLE
Updating Garden View to fit current design

### PR DESCRIPTION
**Pull request by curtisahuang**
## Description

- [X] Description toggle function
- [X] Implement toggle
- [X] Renamed to Daily Gardening
- [X] Removed Garden Box
- [X] Add Image as replacement
- [x] Rename Routes/pages/to fit new name

## Closes issue(s)

If there any issue that is fixed with this PR, please attach it/them here...

## Screenshots

withdetails

## Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [x] I have tested this code

### CODE REVIEW

**Review by kome12 — APPROVED** [review]
LGTM!

File: src/pages/DailyGardening/index.tsx [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-import { Header } from "../../components/Header";
 import { LoadingWrapper } from "../../components/LoadingWrapper";
 import { CompletedTask } from "../../models/completedTask.model";
 import { Rule } from "../../models/rule.model";
 import { useUserState } from "../../store/user/useUserState";
-import "./GardenView.css";
+import "./DailyGardening.css";
 
-export const GardenView = () => {
+export const DailyGardening = () => {
```
Comment (daiHash) [inline src/pages/DailyGardening/index.tsx:20]: 👍

**Review by daiHash — APPROVED** [review]
Nice LGTM!

**Review by conalli — APPROVED** [review]
lgtm

**Pull request merged**